pkg/query-service/app/logparsingpipeline/collector_config.go

```diff
@@ -192,7 +191,7 @@ func GenerateCollectorConfigWithPipelines(
 			))
 		}
 		escapedSerializedConf := strings.ReplaceAll(
-			string(serializedProcConf), "$", "$$$",
+			string(serializedProcConf), "$", "$$",
```

Comment:
the removal of `expandconverter` https://github.com/SigNoz/signoz-otel-collector/pull/617/files#diff-012af3d38a9075715cf23b69501bf7ac9556bfa7db4d463633d208abd6b2df82L205

Reply:
https://github.com/SigNoz/charts-saas-v3/pull/1752/files